pom.xml

```diff
@@ -32,6 +32,8 @@
         <solr.version>4.10.4</solr.version>
         <jena.version>2.13.0</jena.version>
         <slf4j.version>1.7.14</slf4j.version>
+        <reload4j.version>1.2.18.3</reload4j.version>
```

Comment:
Should we target the latest release, 1.2.20?

Reply:
I haven't had a chance to test that personally, but it sounds like a good idea

Reply:
done!